app/helpers/ember_cli_rails_assets_helper.rb

```diff
@@ -0,0 +1,23 @@
+require "ember_cli/assets/lookup"
+
+module EmberCliRailsAssetsHelper
+  def include_ember_script_tags(name, **options)
+    EmberCli[name].build
+
+    assets = EmberCli::Assets::Lookup.new(EmberCli[name])
+
+    assets.javascript_assets.
+      map { |src| %{<script src="#{src}"></script>}.html_safe }.
+      reduce(&:+)
+  end
+
+  def include_ember_stylesheet_tags(name, **options)
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.